mmaction/datasets/pipelines/__init__.py

```diff
@@ -1,13 +1,12 @@
 from .augmentations import (AudioAmplify, CenterCrop, ColorJitter,
-                            EntityBoxClip, EntityBoxCrop, EntityBoxFlip,
-                            EntityBoxPad, EntityBoxRescale, Flip, Fuse,
-                            MelSpectrogram, MultiGroupCrop, MultiScaleCrop,
-                            Normalize, RandomCrop, RandomRescale,
-                            RandomResizedCrop, RandomScale, Resize, TenCrop,
-                            ThreeCrop)
+                            EntityBoxCrop, EntityBoxFlip, EntityBoxRescale,
```

Comment:
Why delete `EntityBoxClip`?

Reply:
Since I handle boxclip in each EntityBoxCrop. After cropping, I will clip the box to make sure that the cropped boxes are in the cropped image.